target-platform/config/kura.target-platform.build.properties

```diff
@@ -13,7 +13,9 @@ org.eclipse.kura.windows.system.version=1.0.0-SNAPSHOT
 org.eclipse.kura.windows.launcher.version=1.0.0-SNAPSHOT
 org.eclipse.kura.windows.service.version=1.0.0-SNAPSHOT
 org.moka7.version=1.0.100
-org.apache.activemq.artemis.version=2.5.0
+
+org.apache.activemq.artemis.version=2.5.100-SNAPSHOT
```

Comment:
Why not having this as 2.5.0-SNAPSHOT?

Reply:
Because version 2.5.0 has been pushed out. So the next version would be 2.5.100-SNAPSHOT from my understanding.

Reply:
2.5.0 has never been pushed out officially. So we can have EMO to clean the repo and work with SNAPSHOt and finally have the official 2.5.0 pushed to the repo when we will release Kura 4.0.0. What do you think?

Reply:
If you prefer that. I will change the PR and get in contact with EMO.

Reply:
Thanks.